quilt/tools/store.py

```diff
@@ -74,27 +76,118 @@ def __init__(self, user, package, mode):
         self._user = user
         self._package = package
         self._mode = mode
-        self._path = self.get_path()
+        self._pkg_dir = None
+        self._path = None
+        self._find_path_read()
 
     def __enter__(self):
         return self
 
     def __exit__(self, type, value, traceback):
         pass
 
+    def get_contents(self):
+        """
+        Returns a dictionary with the contents of the package.
+        """
+        contents = {}
+        try:
+            with open(self._path, 'r') as contents_file:
+                contents = json.loads(contents_file.read())
+        except IOError:
+            # TODO: Should we initialize contents.json on pkg creation?
```

Comment:
So far, we're handling that in command.py. It might be cleaner to make the create/overwrite vs read distinction in store.py, but we want to make sure that we support building packages incrementally.